src/Tribe/Attendees_Table.php

```diff
@@ -224,6 +230,54 @@ public function column_ticket( $item ) {
 	}
 
 	/**
+	 * Generates and returns the attendee table row actions.
+	 *
+	 * @param array $item
+	 *
+	 * @return string
+	 */
+	protected function get_row_actions( array $item ) {
+		/**
+		 * Sets the row action links that display within the ticket column of the
+		 * attendee list table.
+		 *
+		 * @param array $row_actions
+		 * @param array $item
+		 */
+		$row_actions = (array) apply_filters( 'event_tickets_attendees_table_row_actions', array(), $item );
+		$row_actions = join( ' | ', $row_actions );
+		return empty( $row_actions ) ? '' : '<div class="row-actions">' . $row_actions . '</div>';
+	}
+
+	/**
+	 * Returns the attendee ID (or "unique ID" if set).
+	 *
+	 * @param array $item
+	 *
+	 * @return int|string
+	 */
+	public function get_attendee_id( $item ) {
+		$attendee_id = empty( $item['attendee_id'] ) ? '' : $item['attendee_id'];
+		if ( $attendee_id === '' ) {
+			return '';
+		}
+
+		$unique_id = get_post_meta( $attendee_id, '_unique_id', true );
+
+		if ( $unique_id === '' ) {
+			$unique_id = $attendee_id;
+		}
+
+		/**
+		 * Filters the ticket number; defaults to the ticket unique ID.
+		 *
+		 * @param string $unique_id A unique string identifier for the ticket.
+		 * @param array  $item      The item entry.
+		 */
+		return apply_filters( 'tribe_events_tickets_attendees_table_attendee_id_column', $unique_id, $item );
```

Comment:
I almost with we could find a way to make this filter name 45 characters longer.